.buildkite/scripts/release.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- fetching tags"
+# Ensure that tags are present so goreleaser can build the changelog from the last release.
+git rev-parse --is-shallow-repository
+git fetch origin --tags
+
+echo "--- running goreleaser"
+# Run latest version of goreleaser
+# TODO Remove check command once this stage is removed from Jenkinsfile
+# curl -sL https://git.io/goreleaser | bash
+curl -sL https://git.io/goreleaser | bash -s -- check
```

Comment:
As the release step is performed in Jenkins currently, this just performs a check.

Should we remove the release step from jenkins in this PR and enable it here ?

Reply:
I guess we should move it to buildkite at some moment, so feel free to do it now if easy.

Reply:
Ok, I'll remove the release step from Jenkins in favor of this one (removing check flag).